bots/zephyr_mirror_backend.py

```diff
@@ -1004,6 +1041,7 @@ def parse_args():
     return parser.parse_args()
 
 def die_gracefully(signal, frame):
+    # type: (Union[list, None, object], FrameType) -> None
```

Comment:
Sorry, they should all be IntEnums (or Union[Singals, Handlers, Sigmasks]
See https://github.com/python/cpython/blob/c30098c8c6014f3340a369a31df9c74bdbacc269/Lib/signal.py